Sources/Sentry/SentryClient.m

```diff
@@ -355,6 +341,22 @@ - (SentryId *)captureEvent:(SentryEvent *)event
         additionalEnvelopeItems:additionalEnvelopeItems];
 }
 
+- (SentryId *)captureErrorEvent:(SentryEvent *)event
```

Comment:
I didn't add tests for that method because it's covered by the existing tests of `captureException` and `captureError`.